src/rules/preferFunctionOverMethodRule.ts

```diff
@@ -66,10 +66,11 @@ class Walker extends Lint.RuleWalker {
                 const { name } = node as ts.MethodDeclaration;
                 const usesThis = this.withThisScope(
                     name.kind === ts.SyntaxKind.Identifier ? name.text : undefined,
-                    () => super.visitNode(node));
-                if (!usesThis &&
-                    node.parent!.kind !== ts.SyntaxKind.ObjectLiteralExpression &&
-                    this.shouldWarnForModifiers(node as ts.MethodDeclaration)) {
+                    () => super.visitNode(node),
+                );
+                if (!usesThis
+                        && node.parent!.kind !== ts.SyntaxKind.ObjectLiteralExpression
```

Comment:
or maybe a new option in the `align` rule? I agree, just haven't gotten around to filing this feature request